src/panasonicmn_int.cpp

```diff
@@ -720,6 +720,71 @@ namespace Exiv2 {
         return os;
     }  // PanasonicMakerNote::printPitchAngle
 
+/**
+  Chromatic Aberrations info is makernote tag 0x011B. There are 64 bytes of data.
+  The meaning is yet unknown.
+*/
+    std::ostream& PanasonicMakerNote::printChromaticAberration(std::ostream& os, const Value& value, const ExifData*)
+    {
+        std::ostringstream oss;
+        oss.copyfmt(os);
+        for (int i=0; i<value.size(); i++) {
+          os << std::uppercase << std::setfill('0') << std::setw(2) << std::hex << value.toLong(i);
+          os << " ";
+        }
+        os.copyfmt(oss);
+
+        return os;
+    }  // PanasonicMakerNote::printChromaticAberration
+
+/**
+  Distortion Info is makernote tag 0x0119. There are 32 bytes of data. Parsed as 15 signed little endian ints.
+  explanation see: https://syscall.eu/#pana
+  formular: Ru = scale*(Rd + a*Rd^3 + b*Rd^5 + c*Rd^7)
+  exiv2 -g "Exif.PanasonicRaw.DistortionInfo" P1000275.RW2
+  exiv2 -Ph -g "Exif.PanasonicRaw.DistortionInfo" P1000275.RW2
+*/
+    std::ostream& PanasonicMakerNote::printDistortionInfo(std::ostream& os, const Value& value, const ExifData*)
```

Comment:
Also, pretty printing is maybe not the best way to parse this data - perhaps it should be better done as a dedicated data sub-structure/dictionary, like we do e.g. for Nikon MakerNote sub-fields, like "Nikon3 Picture Control Tag Info" etc. (see nikonmn_int.cpp and nikonmn_int.hpp)

Reply:
Yes, I'll do that.

Reply:
> But if I use without "-Ph" it is expected to give human readable information.

Sure, but not necessarily through the pretty print function. If you know the data structure of the blob, then we should probably create it rather than just print it.